app/jobs/clean_all_problems_job.rb

```diff
@@ -0,0 +1,8 @@
+class CleanAllProblemsJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(app_id)
+    app = App.find_by(id: app_id)
+    ::ProblemDestroy.execute(app.problems)
```

Comment:
Because without it you get this error on log:
A copy of xxx has been removed from the module tree but is still active
a quick search on google of problem will give this answer:
http://stackoverflow.com/questions/29636334/a-copy-of-xxx-has-been-removed-from-the-module-tree-but-is-still-active (see  Alex Moore-Niemi reply). I tried it and it worked.